node/src/chain/acceptor.rs

```diff
@@ -234,16 +235,31 @@ impl<DB: database::DB, VM: vm::VMExecution, N: Network> Acceptor<N, DB, VM> {
         .await?;
 
         // Define new block label
-        let label = {
-            let mut label = Label::Accepted;
-            if nil_quorum == Some(true) || blk.header().iteration == 0 {
-                label = Label::Attested;
-                if mrb.is_final() {
-                    label = Label::Final;
-                }
+        let label = match (attested, mrb.is_final()) {
```

Comment:
Can we add a tag in `block accepted` log entry indicating that a block was marked as `Final` due to `rolling finality`?

Reply:
I can add it, but the information is already present there, since the user can check the label of the previous accepted block